commit 9d338424df968670df421d0a910dd941775c4019 (refs/changes/30/2130/1)
Author: Nick Zivkovic <nick.zivkovic@joyent.com>
Date:   2017-06-20T07:53:29+00:00 (2 years, 4 months ago)
    
    TOOLS-1800 Warden should have restify as a peer and dev dependency

diff --git a/package.json b/package.json
index 3c44616..ca33abb 100644
--- a/package.json
+++ b/package.json
@@ -20,6 +20,7 @@
     "eslint": "2.13.1",
     "eslint-plugin-joyent": "1.0.1",
     "tape": "^4.2.2",
+    "restify": "^4.0.0",
     "faucet": "0.0.1"
   },
   "scripts": {
@@ -28,7 +29,9 @@
     "assert-plus": "0.1",
     "vasync": "1.6.4",
     "verror": "1.3",
-    "ip6addr": "0.2.2",
-    "restify": "git://github.com/mcavage/node-restify.git#0d7b4ba"
+    "ip6addr": "0.2.2"
+  },
+  "peerDependencies": {
+    "restify": "^4.0.0"
   }
 }
